Dockerfile

```diff
@@ -4,8 +4,10 @@ EXPOSE 8080
 
 ADD sendEmail-epel-7.repo /etc/yum.repos.d/
 
+ENV NAGIOS_PLUGINS /opt/rhmap/nagios/plugins/
```

Comment:
(optional)
I'd recommend using the syntax with an equal sign, that allows us to easily add more variables later.
https://docs.docker.com/engine/reference/builder/#/env

```Dockerfile
ENV NAGIOS_PLUGINS=/opt/rhmap/nagios/plugins/ \
    FOO=bar
```